tests/e2e/run-scanner-v4-install.bats

```diff
@@ -58,13 +58,16 @@ current_label=""
 post_processor_pid=""
 
 _begin() {
+    local label="${1:-}"
+
     # In case it is convenient for a given test-case, _begin() can take care of the initialization.
     if [[ $initialized = "false" ]]; then
```

Comment:
nit and unrelated to this PR -  would it make sense to have a similar check in `init`? Something like
```
   init() {
    if [[ $initialized = "true" ]]; then
        return 0
    fi
    initialized="true"
```

As it is right now, `_begin()` will only call `init` once, but `init` will be called multiple times (by the test cases themselves).